rest-client/src/test/java/org/acme/restclient/CountriesResourceTest.java

```diff
@@ -23,4 +23,17 @@ public void testCountryNameEndpoint() {
              );
     }
 
-}
\ No newline at end of file
+    @Test
+    public void testCountryNameAsyncEndpoint() {
+        given()
+          .when().get("/country/name/greece")
```

Comment:
You are not calling the right endpoint here? Shouldn't it be name-async?

Reply:
Good catch. It has been updated.